main.go

```diff
@@ -19,11 +19,11 @@ package main
 import (
 	"flag"
 	"fmt"
-
-	"github.com/OJ/gobuster/libgobuster"
+	"./libgobuster"
```

Comment:
This should be changed back I believe, to follow 'proper' standards

Reply:
yep